src/appcallback.rs

```diff
@@ -0,0 +1,66 @@
+use std::marker::Sized;
```

Comment:
I believe this should already be in scope so you don’t need to explicitly import